webapp/src/index.tsx

```diff
@@ -27,7 +30,7 @@ export default class Plugin {
             await store.dispatch(getProviderConfiguration());
 
             registry.registerChannelHeaderMenuAction(
-                <span><i className='icon fa fa-calendar-plus-o'/>{'Create calendar event'}</span>,
+                <span><Icon image={CalendarOutlineIcon}/>{'Create calendar event'}</span>,
```

Comment:
This line is getting kind of long

```suggestion
                (
                    <span>
                        <Icon image={CalendarOutlineIcon}/>
                        {'Create calendar event'}
                    </span>
                ),
```

0/5 on the parens that I added there, but definitely think it should be multiple lines